### ISSUE TITLE
[Fixed in GH-6052] [after #5148] Check focus state in Safari

### ISSUE THREAD

**Issue by kacperknapik** [SAP/cloud-commerce-spartacus-storefront#5175]

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.

**Comment by KateChuen** [comment]
Tested indirectly though other tickets.
